### DISCUSSION TITLE
Security bug report

### DISCUSSION THREAD

**Discussion by ping2327** [clearpool-finance/community#13] [Bug reports]
See the full details of the vulnerability below in the following docs link
[Vulnerability Details](https://docs.google.com/document/d/1EBlssvN0h_X3g53qmR-cv97SwGNSby7tghfcvf_OejE/edit?usp=sharing)

**Comment by ping2327** [comment]


**Reply by georgeciubotaru** [reply]
Hi @ping2327, Thank you for your report.

**Reply by zolotokrylin** [reply]
Hi @ping2327 you are entitled to the reward.
Please share your wallet address. We will airdrop your reward.

**Reply by ping2327** [reply]
Hi @georgeciubotaru and @zolotokrylin , thank you for the quick validation! I appreciate it. You can find my wallet address for the reward in this  [docs link](https://docs.google.com/document/d/10Pb9OR12cCcOjUVusM-gJXBzJJUQRtNARKHCo57SVMQ/)

**Reply by zolotokrylin** [reply]
We are processing the payment. We will let you know when it is done. @ping2327

**Reply by zolotokrylin** [reply]
@ping2327 https://etherscan.io/tx/0x249801bcdbaee9ed9c86b6b4f73db669035a53740596a42379f28800b19df796
